mitiq/tests/test_conversions.py

```diff
@@ -103,6 +131,14 @@ def test_from_mitiq(to_type):
     assert input_type == to_type
 
 
+def test_register_from_mitiq():
+    circuit = qasm_circuit
+    register_mitiq_converter(circuit.__module__, "to", to_qasm)
+    converted_circuit, input_type = convert_to_mitiq(circuit)
+    assert _equal(converted_circuit, cirq_circuit)
+    assert input_type in circuit.__module__
```

Comment:
I'm confused by this test. If we are using `to_qasm`, shouldn't `converted_circuit` be of type `str`? Should line 137 be `convert_from_mitiq`?

Reply:
I made more explicit the use of the mock class in the update: `CircuitStr(str)` takes a string, and defines a `__module_="qasm"` property on it`